### DISCUSSION TITLE
Rename/move multiple pages at once

### DISCUSSION THREAD

**Discussion by tender-barbarian** [requarks/wiki#6772] [General]
Hi,

What about feature proposed here? https://js.wiki/feedback/p/movingrenaming-multiple-pages

Seems like a simple fix that would add a lot of usability... currently if I need to move pages around I have to edit each one individually which becomes a chore, especially since it requires multiple clicks to get to place where page path can be edited.

Cheers!

**Comment by mittkkoo** [comment]
Like many others I'm waiting for v3, where this feature will definitely be implemented. For now I got used to always reference pages by their ID. Like `.../i/123`. And for the navigation bar I use the manual one. By doing so you can completely get rid off the hierarchy and you don't have to move anything. 

For now the easiest way to do it at once would be to enter the database and write a PostgreSQL statement. I found that ChatGPT is quite good at writing these statements for you.
